packages/jest-runtime/src/index.js

```diff
@@ -30,21 +30,21 @@ import {options as cliOptions} from './cli/args';
 
 type Module = {|
   children: Array<Module>,
-  exports: any,
-  filename: string,
-  id: string,
-  loaded: boolean,
-  parent?: Module,
-  paths?: Array<Path>,
-  require?: (id: string) => any,
+    exports: any,
```

Comment:
please run eslint to fix all these whitespace changes

Reply:
i fixed every whitespace by hand.. i missed this file probably or it was overwritten later. i will fix it.

Reply:
Hmm, you shouldn't have to. Just `yarn eslint . --fix` should be enough. I don't have issues with the lint integration, but I don't use VS code either